wpimath/src/main/native/include/frc/geometry/Twist3d.h

```diff
@@ -50,6 +52,25 @@ struct WPILIB_DLLEXPORT Twist3d {
    */
   units::radian_t rz = 0_rad;
 
+
+  /**
+   * Obtain a new Transform3d from a (constant curvature) velocity.
+   *
+   * <p>See <a href="https://file.tavsys.net/control/controls-engineering-in-frc.pdf">Controls
```

Comment:
JavaDoc needs `<p>` tags in .java files, but Doxygen doesn't need `<p>` tags in C++ files, so they should be removed from the latter.